elkscmd/ash/Makefile

```diff
@@ -46,8 +46,9 @@ ash: $(OBJS)
 
 install: ash
 ifdef CONFIG_IMG_LINK
-	$(INSTALL) ash $(DESTDIR)/bin
-	$(LN) -sf ash $(DESTDIR)/bin/sh
+#	$(INSTALL) ash $(DESTDIR)/bin
+#	$(LN) -sf ash $(DESTDIR)/bin/sh
+	$(INSTALL) ash $(DESTDIR)/bin/sh
```

Comment:
Useless 'ifdef' after the change. To be simplified.

Reply:
OK